gulpfile.js

```diff
@@ -285,6 +286,7 @@ const criticalCssConfig = (function () {
     return {
         baseUrl: 'http://localhost:8080',
         baseFilePath: './build/critical-css',
+        assetPathPrefix: '/assets/patterns/level-to-be-raised-from-by-actual-path-double-dot/',
```

Comment:
I'd move the fake part to `critical.generate()`, since it's a hack in a specific usage.